Scripts/Toolbox/Editor/DialogueImporter.cs

```diff
@@ -378,5 +466,61 @@ private Character FindCharacterInDB(string name, out int index)
             }
             return found;
         }
+
+        
+        private Expression FindExpression(string exName, Character character, out int exNum)
+        {
+            
+            for(int i = 0; i < database.Characters.Count; i++)
+            {
+                if(database.Characters[i].FirstName == character.FirstName)
+                {
+                    for(int en = 0; en < database.Characters[i].Expressions.Count; en++)
+                    {
+                        if (exName == database.Characters[i].Expressions[en].Name)
+                        {
+                            exNum = en+1;
+                            Debug.Log(en);
+                            Debug.Log("POGGERS " + database.Characters[i].Expressions[en].Name);
+                            return database.Characters[i].Expressions[en];
+                        }
+                    }
+                }
+            }
+            /* go into the characterdatabase.character and find that characters list of expressions
+               via a foreach loop and if it equals one of the expressions then return that expression */
+            exNum = 0;
+            return null;
+        }
+        private AudioClip FindVoice(string voiceName)
+        {
+            //Look through the resource folder to find the voice file and return it
```

Comment:
Comment not needed (make it self documenting instead). If you wanted to highlight that it's done in the Resources folder, then something like `FindVoiceInResources` would make that clear w/o comment.